server/controllers/user.js

```diff
@@ -2,10 +2,12 @@ import isEmpty from "lodash.isempty";
 import database from "../models/index";
 import { validateData } from "../validations/validateData";
 import { signInSchema } from "../validationSchemas/user";
-import { generateJWT } from "../helpers/jwt";
 import { comparePassword } from "../helpers/password";
-import { httpResponse } from "../helpers/http";
+import { httpResponse, serverError } from "../helpers/http";
+import { getBaseUrl } from "../helpers/index";
+import userResponse from "../helpers/userResponse";
 
+const User = database["Users"];
```

Comment:
import Users from models, we should be using this syntax again

Reply:
I will have to refractor the user model to be able to do that. I tried it first, it didn't work, it will break every body work that have used the user model